src/game/savegame/savegame_bson.c

```diff
@@ -822,6 +825,27 @@ static bool SaveGame_BSON_LoadCurrentMusic(struct json_object_s *music_obj)
                 current_track, timestamp);
         }
     }
+    return true;
+}
+
+static bool SaveGame_BSON_LoadMusicTrackFlags(
+    struct json_array_s *music_track_arr)
+{
+    if (!music_track_arr) {
+        LOG_WARNING("Malformed save: invalid or missing music track array");
```

Comment:
Since we chose not to bump the version, this ominous warning message will show in the logs for good saves from the current stable. I think that we either should bump the version after all, or remove the warning. After giving it some more thinking, I'm leaning towards bumping the version to V3 and restoring the conditions for >= V3 for both music and music flags. @lahm86 @walkawayy

Reply:
Bumped version.